F/FastANI/build_tarballs.jl

```diff
@@ -0,0 +1,46 @@
+using BinaryBuilder, Pkg
+
+name = "FastANI"
+version = v"1.33.0"
+sources = [
+    GitSource("https://github.com/ParBLiSS/FastANI.git", "6531da0957344d2a56e3eeb9bb83fea02cc4abe0"),
+]
+
+
+script = raw"""
+cd ${WORKSPACE}/srcdir/FastANI
+
+update_configure_scripts --reconf
+autoupdate
+
+./bootstrap.sh
+./configure \
+    --prefix=$prefix \
+    --build=${MACHTYPE} \
+    --host=${target} \
+    --with-gsl=${destdir} \
+    --includedir=${includedir} \
+    --libdir=${libdir}
+CXXFLAGS="-O3 -DNDEBUG -std=c++11 -Isrc -fopenmp" 
+make CXXFLAGS="${CXXFLAGS}"
+make install
```

Comment:
If I'm reading it correctly, you can just replace all of this with
```suggestion
mkdir -p "${bindir}"
c++ -O3 -DNDEBUG -std=c++11 -Isrc -fopenmp   src/cgi/core_genome_identity.cpp -o "${bindir}/fastANI${exeext}" -lgsl -lgslcblas -lz -lm  
```